onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7584,7 +7587,33 @@ def aten_scatter_reduce(
         self = op.Reshape(self, neg_1)
         index = op.Reshape(index, neg_1)
         src = op.Reshape(src, neg_1)
+
+    if not include_self:
```

Comment:
Could you add a comment to clarify the logic here since I don't find it immediately apparent? Thanks!